snykTask/src/task-args.ts

```diff
@@ -10,7 +14,7 @@ class TaskArgs {
   dockerfilePath: string | undefined = '';
 
   severityThreshold: string | undefined = '';
-
+  failOnThreshold: string | undefined = 'low';
```

Comment:
You don't need `undefined` in type definition because you provide default value for the property.